src/events/notification/index.ts

```diff
@@ -113,6 +114,12 @@ export abstract class Notification implements INotification {
   }
 
   private sanitizeValue(value: string): string {
-    return value.replace(/_/g, ' ');
+    if (typeof value == 'string') {
+      return value.replace(/_/g, ' ');
+    }
+    else 
```

Comment:
You could remove the else block and just return value. It's effectively the same thing but slightly cleaner.